modules/administration/pages/tshoot-rpctimeout.adoc

```diff
@@ -27,30 +27,30 @@ While this will not resolve networking problems, it will cause a process to fail
 
 .Procedure: Resolving RPC connection timeouts
 
-. On the {productname} Server, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds) for this parameter:
+. On the {productname} server, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds):
```

Comment:
Shouldn't Server/Proxy be capped if it comes after {productname}?

Reply:
Sounds like a clear rule ;)

Reply:
another review req, pls